tools/fog-local-network/fog_conformance_tests.py

```diff
@@ -279,17 +283,19 @@ def balance_check(self, step_name, acceptable_answers_per_wallet):
         assert len(acceptable_answers_per_wallet) == self.NUM_WALLETS
 
         print(f'{step_name}: Performing fresh balance checks')
-        new_wallets = [
-            self._fresh_balance_check(step_name, wallet_num, acceptable_balances, expected_eventual_block_count)
-            for wallet_num, (acceptable_balances, expected_eventual_block_count) in enumerate(acceptable_answers_per_wallet)
-        ]
+        # The lambda we will pass to thread-pool
+        # input should be: (wallet_num, (acceptable_answers, expected_eventual_block_count))
+        map_fn = lambda args: self._fresh_balance_check(step_name, args[0], args[1][0], args[1][1])
```

Comment:
This is confusing to look at. At the very least I would consider replacing it with something like this:
```
def map_fn(args): 
    (wallet_num, (acceptable_answers, expected_eventual_block_count)) = args
    self._fresh_balance_check(step_name, wallet_num, acceptable_answers, expected_eventual_block_count)
```

Reply:
thank you, i didn't know this works in python